third_party/move/move-vm/types/src/values/values_impl.rs

```diff
@@ -1466,7 +1466,10 @@ impl IntegerValue {
                 return Err(PartialVMError::new(StatusCode::INTERNAL_TYPE_ERROR).with_message(msg));
             },
         };
-        res.ok_or_else(|| PartialVMError::new(StatusCode::ARITHMETIC_ERROR))
+        res.ok_or_else(|| {
+            PartialVMError::new(StatusCode::ARITHMETIC_ERROR)
+                .with_message("Add overflow".to_string())
```

Comment:
Consider changing this to "Addition" (and similarly, "Subtraction" below) to match with "Multiplication".